### PR TITLE
[DO NOT MERGE] Adding usage id property to support usages by VM family, Workspace & Comput

### PR DESCRIPTION
**Pull request by leizhaoPS**
[DO NOT MERGE] now since the backend still in implementation.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by leizhaoPS** [comment]
@sarangan12 , can you help add "WaitForARMFeedback" label and add those folks as reviewer?
Gaurav Bhatnagar <Gaurav.Bhatnagar@microsoft.com>; Sanjai Ganesh Jeenagiri <sanjaiga@microsoft.com>;Kristopher Bash <krbash@microsoft.com>
Thanks a lot!

File: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/examples/ListUsages.json [hunk 1]
```diff
@@ -2,58 +2,169 @@
   "parameters": {
     "subscriptionId": "00000000-0000-0000-0000-000000000000",
     "location": "eastus",
-    "api-version": "2018-11-19"
+    "api-version": "2019-05-01"
```
Comment (sarangan12) [inline specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/examples/ListUsages.json]: If the api-version is this, why are you adding this file in 2018-11-19/examples folder?

Comment (leizhaoPS) [reply specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/examples/ListUsages.json]: @sarangan12 , good catch, I synced this with team, I will create a new PR with folder "2019-05-01", when it checked-in, I will update usages changes in another PR.
Here is the PR for migrate version without content changes, please help sign-off: https://github.com/Azure/azure-rest-api-specs/pull/6220

**Comment by KrisBash** [comment]
Is PR https://github.com/Azure/azure-rest-api-specs/pull/5910  still needed?

**Comment by leizhaoPS** [comment]
> Is PR #5910 still needed?

yes, I will ask Kalpana abandon https://github.com/Azure/azure-rest-api-specs/pull/5910

**Comment by leizhaoPS** [comment]
Closed this PR, moved the changes to https://github.com/Azure/azure-rest-api-specs/pull/6220

**Pull request closed without merging**